graphql/tests/task/data.json

```diff
@@ -822,524 +824,71 @@
-      "distro": "ubuntu1604-build",
-      "r": "github_pull_request",
-      "display_name": "test-model",
-      "task_create_time": {
-        "$date": "2020-02-20T20:37:06Z"
-      },
-      "test_start_time": {
-        "$date": "2020-02-21T15:15:56Z"
-      },
-      "test_end_time": {
-        "$date": "2020-02-21T15:15:56.009Z"
-      }
-    },
-    {
-      "_id": "5e4ff42d4e88faea7a90f0d2",
-      "status": "pass",
-      "test_file": "TestCreateTaskGroup",
-      "log_id": "5e4ff42d30661528ad67fafb",
-      "line_num": 107,
-      "exit_code": 0,
-      "start": 1582298156,
-      "end": 1582298156,
-      "task_id": "1",
-      "task_execution": 0,
-      "branch": "evergreen",
-      "build_variant": "ubuntu1604",
-      "distro": "ubuntu1604-build",
-      "r": "github_pull_request",
-      "display_name": "test-model",
-      "task_create_time": {
-        "$date": "2020-02-20T20:37:06Z"
-      },
-      "test_start_time": {
-        "$date": "2020-02-21T15:15:56Z"
-      },
-      "test_end_time": {
-        "$date": "2020-02-21T15:15:56Z"
-      }
-    },
-    {
-      "_id": "5e4ff42d4e88faea7a90f0e6",
-      "status": "pass",
-      "test_file": "TestRetryCommitQueueItems",
-      "log_id": "5e4ff42d30661528ad67fafb",
-      "line_num": 203,
-      "exit_code": 0,
-      "start": 1582298156,
-      "end": 1582298156.07,
-      "task_id": "1",
-      "task_execution": 0,
-      "branch": "evergreen",
-      "build_variant": "ubuntu1604",
-      "distro": "ubuntu1604-build",
-      "r": "github_pull_request",
-      "display_name": "test-model",
-      "task_create_time": {
-        "$date": "2020-02-20T20:37:06Z"
-      },
-      "test_start_time": {
-        "$date": "2020-02-21T15:15:56Z"
-      },
-      "test_end_time": {
-        "$date": "2020-02-21T15:15:56.069Z"
-      }
-    },
-    {
-      "_id": "5e4ff42d4e88faea7a90f0ee",
-      "status": "pass",
-      "test_file": "TestProjectAliasSuite/TestInsertTagsAndNoVariant",
-      "log_id": "5e4ff42d30661528ad67fafb",
-      "line_num": 219,
-      "exit_code": 0,
-      "start": 1582298156,
-      "end": 1582298156.02,
-      "task_id": "1",
-      "task_execution": 0,
-      "branch": "evergreen",
-      "build_variant": "ubuntu1604",
-      "distro": "ubuntu1604-build",
-      "r": "github_pull_request",
-      "display_name": "test-model",
-      "task_create_time": {
-        "$date": "2020-02-20T20:37:06Z"
+      "_id": {
+        "task_id": "1",
+        "execution": 0
       },
-      "test_start_time": {
-        "$date": "2020-02-21T15:15:56Z"
+      "stats": {
+        "total_count": 6,
+        "failed_count": 2
       },
-      "test_end_time": {
-        "$date": "2020-02-21T15:15:56.020Z"
-      }
+      "results": [
+        {
+          "test_name": "TestStuckHostAuditing",
+          "task_id": "1",
+          "execution": 0,
+          "status": "fail",
+          "line_num": 19,
+          "test_start_time": "2020-02-21T15:15:56Z",
```

Comment:
Yes, but it doesn't make a difference since it still is able to get unmarshalled into a `time.Time` object in Go. I will update them just to be more explicit.